spring-cloud/spring-cloud-stream/spring-cloud-stream-kafka/src/main/java/com/baeldung/consumer/AvroConsumer.java

```diff
@@ -0,0 +1,21 @@
+package com.baeldung.consumer;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.cloud.stream.annotation.StreamListener;
+import org.springframework.cloud.stream.messaging.Processor;
+import org.springframework.stereotype.Service;
+
+import com.baeldung.schema.Employee;
+
+@Service
+public class AvroConsumer {
+
+    private Logger logger = LoggerFactory.getLogger(this.getClass());

```

Comment:
Should be `private static final Logger LOGGER` and use `AvroConsumer.class`

Reply:
done